android/src/main/java/org/reactnative/camera/RNCameraView.java

```diff
@@ -356,17 +355,10 @@ public void onBarCodeScanningTaskCompleted() {
     }
   }
 
-  // Limit Scan Area
-  public void setRectOfInterest(float x, float y, float width, float height) {
-    this.mLimitScanArea = true;
-    this.mScanAreaX = x;
-    this.mScanAreaY = y;
-    this.mScanAreaWidth = width;
-    this.mScanAreaHeight = height;
-  }
-    public void setCameraViewDimensions(int width, int height) {
-    this.mCameraViewWidth = width;
-    this.mCameraViewHeight = height;
+  public void setRectOfInterest(int dataWidth, int dataHeight, int left, int top, int width, int height) {
```

Comment:
I should test for conflicts with IOS functioanality and consider using a different name for this prop...

Reply:
I need to add a type [here](https://github.com/react-native-community/react-native-camera/blob/5abbacaac4448311775c5544ea5feb3c3b5b94f9/src/RNCamera.js#L285) or change the existing `rectOfInterest`. It is still not clear to me if it is already merged for iOS.

Reply:
so we gonna have both props?

Reply:
@sibelius I am investigating if the functionality was implemented for iOS. Additionally I had made some small changes to the implementation example https://github.com/fabriziobertoglio1987/react-native-camera/commit/ea167f6b8f33ecc37772e618c273d0ae74da68dc to avoid breaking the ios functionalities.

I see many non-merged pull request for this functionality, so maybe it was not implemented for ios. Searching [rectOfInterest](https://github.com/react-native-community/react-native-camera/search?q=rectOfInterest&unscoped_q=rectOfInterest) and [setRectOfInterest](https://github.com/react-native-community/react-native-camera/search?q=setRectOfInterest&unscoped_q=setRectOfInterest) only shows functionality to store the value, but not for using it in `zxing`...

I will check out again https://github.com/react-native-community/react-native-camera/pull/2719 and test that that it works also on iOS and keep searcing...

Maybe rectOfInterest will be removed... Thanks a lot

Reply:
https://github.com/react-native-community/react-native-camera/pull/2209

Reply:
Hopefully in the android code I can detect the screen orientation and pass the props as `float` value from 0 to 1 (a percentage relative to the screen dimension) as in the ios functioanlity and have `rectOnProps` be exactly the same between iOS and Android